widgetssdk/src/main/java/com/glia/widgets/chat/StatefulChatInputBackground.kt

```diff
@@ -0,0 +1,62 @@
+package com.glia.widgets.chat
+
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+import android.view.View
+import androidx.annotation.ColorInt
+import com.glia.widgets.R
+import com.glia.widgets.helper.getAttr
+import com.glia.widgets.helper.getColorCompat
+import com.glia.widgets.messagecenter.SimpleStatefulWidgetAdapter
+import com.glia.widgets.messagecenter.StatefulWidgetAdapter
+import com.glia.widgets.messagecenter.StatefulWidgetAdapterCallback
+import com.glia.widgets.view.unifiedui.applyLayerTheme
+import com.glia.widgets.view.unifiedui.theme.base.ColorTheme
+import com.glia.widgets.view.unifiedui.theme.base.LayerTheme
+
+internal class StatefulChatInputBackground @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null) : View(context, attrs) {
```

Comment:
I would appreciate some comment on why this was added for future reference or you may shortly summarize what this class tries to achive

Reply:
fixed - eccc1faaebbf293da3ecb4927c4fdf3a78aac252